agw/main.tf

```diff
@@ -0,0 +1,78 @@
+locals {
+  resource_group = "${var.environment}_${var.application_name}_gateway"
```

Comment:
Can the resourcegroup stuff be removed? I don't think any of the sub-modules should be responsible for managing or creating the resourcegroup (aside from the vnet), can this be changed to take the input from another Terraform codebase?

Reply:
Ok we can remove this.

Reply:
Commit:
6f5845a